### PR TITLE
added enrolledMentees list api

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: src/services/mentees.js [hunk 1]
```diff
@@ -1042,6 +1056,18 @@ module.exports = class MenteesHelper {
 				additionalProjectionString,
 				false
 			)
+			mentorExtensionDetails = await mentorQueries.getMentorsByUserIdsFromView(
```
Comment (rakeshSgr) [inline src/services/mentees.js:1059]: @nevil-mathew Why was this added multiple times?

Comment (nevil-mathew) [reply src/services/mentees.js:1060]: That is for the pagination logic. This is the existing flow. @rakeshSgr

File: src/services/sessions.js [hunk 2]
```diff
@@ -1698,4 +1698,137 @@ module.exports = class SessionsHelper {
...
+				'last_logged_in_at',
+				'has_accepted_terms_and_conditions',
+				'languages',
+				'preferred_language',
+				'custom_entity_text',
+			]
+
+			const cleanedAttendeesAccounts = mergedUserArray.map((user) => {
+				propertiesToDelete.forEach((property) => {
+					delete user[property]
```
[the remark this answers is not in the document]
Comment (nevil-mathew) [reply src/services/sessions.js:1819]: We can have that, but few of the keys come from the user service. That's why I went with this flow.
